src/vtkjs/vtkjsRenderer.js

```diff
@@ -0,0 +1,181 @@
+ * @returns {geo.gl.vtkjsRenderer}
+ */
+var vtkjsRenderer = function (arg) {
+  'use strict';
+
+  if (!(this instanceof vtkjsRenderer)) {
+    return new vtkjsRenderer(arg);
+  }
+  arg = arg || {};
+  renderer.call(this, arg);
+
+  var mat4 = require('gl-mat4');
+  var geo_event = require('../event');
+  var vtkjs = vtkjsRenderer.vtkjs;
+  var vtkFullScreenRenderWindow = vtkjs.Rendering.Misc.vtkFullScreenRenderWindow;
+
+  var m_this = this,
+      m_width = 0,
+      m_height = 0,
+      s_init = this._init;
+
+  var fullScreenRenderer = vtkFullScreenRenderWindow.newInstance({
+    background: [0.1, 0.5, 0.5] });
+  var vtkjsren = fullScreenRenderer.getRenderer();
+  var renderWindow = fullScreenRenderer.getRenderWindow();
+
+  /**
+   * Return width of the renderer
+   */
+  this.width = function () {
+    return m_width;
+  };
+
+  /**
+   * Return height of the renderer
+   */
+  this.height = function () {
+    return m_height;
+  };
+
+  /**
+   * Get context specific renderer
+   */
+  this.contextRenderer = function () {
+    return vtkjsren;
+  };
+
+  /**
+   * Get API used by the renderer
+   */
+  this.api = function () {
+    return 'vtkjs';
+  };
+
+  /**
+   * Initialize
+   */
+  this._init = function () {
+    if (m_this.initialized()) {
+      return m_this;
+    }
+
+    s_init.call(m_this);
+
+    /* Initialize the size of the renderer */
+    var map = m_this.layer().map(),
+        mapSize = map.size();
+    m_this._resize(0, 0, mapSize.width, mapSize.height);
+    return m_this;
+  };
+
+  /**
+   * Handle resize event
+   */
+  this._resize = function (x, y, w, h) {
+    m_this._render();
+
+    return m_this;
+  };
+
+  /**
+   * Render.  This actually schedules rendering for the next animation frame.
+   */
+  this._render = function () {
+    /* If we are already scheduled to render, don't schedule again.  Rather,
+     * mark that we should render after other animation frame requests occur.
+     * It would be nice if we could just reschedule the call by removing and
+     * readding the animation frame request, but this doesn't work for if the
+     * reschedule occurs during another animation frame callback (it then waits
+     * until a subsequent frame). */
+    m_this.layer().map().scheduleAnimationFrame(this._renderFrame, true);
+    return m_this;
+  };
+
+  /**
+   * This clears the render timer and actually renders.
+   */
+  this._renderFrame = function () {
+    m_this._updateRendererCamera();
+    renderWindow.render();
+  };
+
+  /**
+   * Exit
+   */
+  this._exit = function () {
+    // DO NOTHING
+  };
+
+  this._updateRendererCamera = function () {
+    var map = m_this.layer().map(),
+        camera = map.camera(),
+        view = camera.view,
+        proj = camera.projectionMatrix;
+    var viewmat = mat4.create();
+    mat4.copy(viewmat, view);
+    var projmat = mat4.create();
+    mat4.copy(projmat, proj);
+    m_this.contextRenderer().getActiveCamera().setViewMatrix(viewmat);
```

Comment:
This function `setViewMatrix` [does not exist in vtkjs](https://github.com/Kitware/vtk-js/blob/9063bb4db9da420f5e127d72a9cba60802f6624b/Sources/Rendering/OpenGL/Camera/index.js).

Reply:
It exists in `Rendering/Core/Camera/`. kitware/vtk-js#756 added it.

Reply:
Thanks, I will update our vtk.js dependency.